mon-pix/translations/fr.json

```
@@ -253,7 +253,8 @@
       "fields": {
         "cgu": {
           "error": "Vous devez accepter les conditions d’utilisation de Pix pour créer un compte.",
-          "label": "'<abbr title=\"obligatoire\">'*'</abbr>' J'accepte les '<a href=\"{cguUrl}\" class=\"link\" target=\"_blank\">'conditions d'utilisation de Pix'</a>'"
+          "label": "'<abbr title=\"obligatoire\">'*'</abbr>' J'accepte les ",
```

@MelanieMEB pour le futur : on pouvait aussi corriger le bug en déplaçant l'apostrophe qui termine la chaîne échappée commençant par `<a href="` de façon à ce qu'elle termine la chaîne échappée _avant_ l'apparition de `{cguUrl}` (au lieu que la chaîne inclue tout le tag ouvrant `<a>` comme c'était le cas).

```suggestion
          "label": "'<abbr title=\"obligatoire\">'*'</abbr>' J'accepte les '<a href=\"'{cguUrl}\" class=\"link\" target=\"_blank\">conditions d'utilisation de Pix'</a>'"	
```

J'ai un peu galéré à comprendre les règles d'échappement dans les messages ICU, du coup je me suis fait un petit notebook de test : https://runkit.com/jonathanperret/5f1ea744a94c3b001bcb0921 . Le piège principal c'est que l'apostrophe ne commence une chaîne échappée que si elle précède un caractère nécessitant un échappement (`{` et `<` essentiellement) !